conformance/conformance_test.go

```diff
@@ -86,3 +89,15 @@ func parseSupportedFeatures(f string) sets.Set[suite.SupportedFeature] {
 	}
 	return res
 }
+
+func parseNamespaceLabels(f string) map[string]string {
```

Comment:
Mind adding a simple unit test for this?

Reply:
Agree these are trivial unit tests, but we've had small things sit broken in conformance for longer than I'd like due to lack of test coverage. I'd rather lean towards spending the extra time here to avoid future pain.

Reply:
done